config/webpack.config.js

```diff
@@ -71,6 +71,12 @@ module.exports = env => {
   if (env && env.pluginName !== undefined) {
     var pluginEntries = {};
     pluginEntries[env.pluginName] = plugins['entries'][env.pluginName];
+    for (var entry in plugins['entries']) {
```

Comment:
`for-in` is a bad practice in javascript, you can read more about it here:
https://hacks.mozilla.org/2015/04/es6-in-depth-iterators-and-the-for-of-loop/

Can you migrate to `for-of`?

Reply:
Thanks, my js knowledge is fairly out of date :)